packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -387,6 +387,17 @@ export class MgtPerson extends MgtTemplatedComponent {
   })
   public personCardInteraction: PersonCardInteraction;
 
+  /**
+   * Get the scopes required for person
+   *
+   * @static
+   * @return {*}  {string[]}
+   * @memberof MgtPerson
+   */
+  public static getScopes(): string[] {
+    return ['contacts.read', 'user.readbasic.all', 'user.read', 'people.read', 'presence.read.all'];
```

Comment:
Similar to how the person card returns scopes based on the config, the person component should do the same for the `contacts.read` - they should only be returned if contact apis are enabled in the config. 

Also missing `presence.read` - it is used if the query is `me`.